Index: src/test/java/runners/Runner.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/test/java/runners/Runner.java	(revision 625803b998e2fe811331dacad2558ccc21c0c89c)
+++ src/test/java/runners/Runner.java	(date 1605127342854)
@@ -9,7 +9,7 @@
                 "json:target/json-reports/cucumber.json",
                 "junit:target/xml-report/cucumber.xml"
         },
-        features = "/Users/administrator/IdeaProjects/com.gmibank/src/test/resources/features/US3passwordCreate.feature",
+        features = "/Users/administrator/IdeaProjects/com.gmibank/src/test/resources/features/US03passwordCreate.feature",
         glue = "stepdefinitions",
         tags = "@US03",
         dryRun = false
diff --git src/test/resources/features/US3passwordCreate.feature src/test/resources/features/US03passwordCreate.feature
rename from src/test/resources/features/US3passwordCreate.feature
rename to src/test/resources/features/US03passwordCreate.feature
