Index: configuration.properties
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>browser = chrome\ntoken   = eyJhbGciOiJIUzUxMiJ9.eyJzdWIiOiJhZG1pbiIsImF1dGgiOiJST0xFX0FETUlOLFJPTEVfVVNFUiIsImV4cCI6MTYwNDYxNzgyM30.6pYwvDKGECkHg6dx5Z__mYZofyHGutdyAw42IRaDNwiyxNtFWUmpTimfh1r3DklSrYqkXUaSVikcWH_ooBd5UA\napi_url = https://www.gmibank.com/api/tp-account-registrations\napi=new\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- configuration.properties	(revision 13aefb23be1a0d12e86a88eb4606005181cfdfa0)
+++ configuration.properties	(date 1604772543773)
@@ -1,4 +1,3 @@
 browser = chrome
 token   = eyJhbGciOiJIUzUxMiJ9.eyJzdWIiOiJhZG1pbiIsImF1dGgiOiJST0xFX0FETUlOLFJPTEVfVVNFUiIsImV4cCI6MTYwNDYxNzgyM30.6pYwvDKGECkHg6dx5Z__mYZofyHGutdyAw42IRaDNwiyxNtFWUmpTimfh1r3DklSrYqkXUaSVikcWH_ooBd5UA
-api_url = https://www.gmibank.com/api/tp-account-registrations
-api=new
+api_url = https://www.gmibank.com/api/tp-account-registrations
\ No newline at end of file
Index: src/test/resources/features/sample.feature
===================================================================
--- src/test/resources/features/sample.feature	(revision 13aefb23be1a0d12e86a88eb4606005181cfdfa0)
+++ src/test/resources/features/sample.feature	(revision 13aefb23be1a0d12e86a88eb4606005181cfdfa0)
@@ -1,7 +0,0 @@
-# new feature
-# Tags: optional
-
-Feature: A description
-
-  Scenario: A scenario
-    Given something..
\ No newline at end of file
Index: src/test/java/stepdefinitions/SampleDef.java
===================================================================
--- src/test/java/stepdefinitions/SampleDef.java	(revision 13aefb23be1a0d12e86a88eb4606005181cfdfa0)
+++ src/test/java/stepdefinitions/SampleDef.java	(revision 13aefb23be1a0d12e86a88eb4606005181cfdfa0)
@@ -1,4 +0,0 @@
-package stepdefinitions;
-
-public class SampleDef {
-}
Index: src/test/java/pages/samplePage.java
===================================================================
--- src/test/java/pages/samplePage.java	(revision 13aefb23be1a0d12e86a88eb4606005181cfdfa0)
+++ src/test/java/pages/samplePage.java	(revision 13aefb23be1a0d12e86a88eb4606005181cfdfa0)
@@ -1,4 +0,0 @@
-package pages;
-
-public class samplePage {
-}
Index: src/test/java/runners/Runner.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package runners;\nimport io.cucumber.junit.Cucumber;\nimport io.cucumber.junit.CucumberOptions;\nimport org.junit.runner.RunWith;\n\n@RunWith(Cucumber.class)\n@CucumberOptions(\n        plugin = {\"html:target/default-cucumber-reports\",\n                \"json:target/json-reports/cucumber.json\",\n                \"junit:target/xml-report/cucumber.xml\"\n        },\n        features = \"src/test/resources/features\",\n        glue = \"stepdefinitions\",\n        tags = \"@GLB110GT017-01\",\n        dryRun = false\n)\n\npublic class Runner {\n}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/test/java/runners/Runner.java	(revision 13aefb23be1a0d12e86a88eb4606005181cfdfa0)
+++ src/test/java/runners/Runner.java	(date 1604772403761)
@@ -11,7 +11,7 @@
         },
         features = "src/test/resources/features",
         glue = "stepdefinitions",
-        tags = "@GLB110GT017-01",
+        tags = "@US01TC01",
         dryRun = false
 )
 
Index: src/test/java/stepdefinitions/ManageCostumerStepDef.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/test/java/stepdefinitions/ManageCostumerStepDef.java	(date 1604772403762)
+++ src/test/java/stepdefinitions/ManageCostumerStepDef.java	(date 1604772403762)
@@ -0,0 +1,48 @@
+package stepdefinitions;
+
+import com.sun.tools.jxc.ConfigReader;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import utilities.ConfigurationReader;
+import utilities.Driver;
+
+public class ManageCostumerStepDef {
+
+    @Given("Go to {string} link")
+    public void goToLink(String arg0) {
+        Driver.getDriver().get(ConfigurationReader.getProperty("gmi_url"));
+    }
+
+    @And("Click to User drop down menu")
+    public void clickToUserDropDownMenu() {
+    }
+
+    @And("Click to Sign in")
+    public void clickToSignIn() {
+    }
+
+    @And("Enter the username with valid credentail")
+    public void enterTheUsernameWithValidCredentail() {
+    }
+
+    @And("Enter the pasword with valid credentail")
+    public void enterThePaswordWithValidCredentail() {
+    }
+
+    @And("Click to Sign in button")
+    public void clickToSignInButton() {
+    }
+
+    @And("Click to My Operations drop down menu")
+    public void clickToMyOperationsDropDownMenu() {
+    }
+
+    @And("Click to Manage Costumers")
+    public void clickToManageCostumers() {
+    }
+
+    @Then("Random customer should show up on manage customers module populating the account information of the customer.")
+    public void randomCustomerShouldShowUpOnManageCustomersModulePopulatingTheAccountInformationOfTheCustomer() {
+    }
+}
Index: src/test/resources/features/manageCostumer.feature
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/test/resources/features/manageCostumer.feature	(date 1604772403762)
+++ src/test/resources/features/manageCostumer.feature	(date 1604772403762)
@@ -0,0 +1,14 @@
+@US12TC01
+
+Feature: A description
+
+  Scenario: A scenario
+Given Go to "https://gmibank.com/" link
+And Click to User drop down menu
+And Click to Sign in
+And Enter the username with valid credentail
+And Enter the pasword with valid credentail
+And Click to Sign in button
+And Click to My Operations drop down menu
+And Click to Manage Costumers
+Then Random customer should show up on manage customers module populating the account information of the customer.
Index: src/test/java/pages/ManageCostumerPage.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/test/java/pages/ManageCostumerPage.java	(date 1604772403761)
+++ src/test/java/pages/ManageCostumerPage.java	(date 1604772403761)
@@ -0,0 +1,4 @@
+package pages;
+
+public class ManageCostumerPage {
+}
