Index: src/test/java/runners/Runner.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package runners;\nimport io.cucumber.junit.Cucumber;\nimport io.cucumber.junit.CucumberOptions;\nimport org.junit.runner.RunWith;\n\n@RunWith(Cucumber.class)\n@CucumberOptions(\n        plugin = {\"html:target/default-cucumber-reports\",\n                \"json:target/json-reports/cucumber.json\",\n                \"junit:target/xml-report/cucumber.xml\"\n        },\n        features = \"src/test/resources/features\",\n        glue = \"stepdefinitions\",\n        tags = \"@US01TC01\",\n        dryRun = false\n)\n\npublic class Runner {\n}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/test/java/runners/Runner.java	(revision 8177804d26826c245f3356780f45575d7ecba8b4)
+++ src/test/java/runners/Runner.java	(date 1605080717901)
@@ -9,9 +9,9 @@
                 "json:target/json-reports/cucumber.json",
                 "junit:target/xml-report/cucumber.xml"
         },
-        features = "src/test/resources/features",
+        features = "/Users/air/IdeaProjects/com.gmibank/src/test/resources/features",
         glue = "stepdefinitions",
-        tags = "@US01TC01",
+        tags = "@US005TC01",
         dryRun = false
 )
 
Index: configuration.properties
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>browser = chrome\ntoken   = eyJhbGciOiJIUzUxMiJ9.eyJzdWIiOiJhZG1pbiIsImF1dGgiOiJST0xFX0FETUlOLFJPTEVfVVNFUiIsImV4cCI6MTYwNDYxNzgyM30.6pYwvDKGECkHg6dx5Z__mYZofyHGutdyAw42IRaDNwiyxNtFWUmpTimfh1r3DklSrYqkXUaSVikcWH_ooBd5UA\napi_url = https://www.gmibank.com/api/tp-account-registrations
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- configuration.properties	(revision 8177804d26826c245f3356780f45575d7ecba8b4)
+++ configuration.properties	(date 1605021767173)
@@ -1,3 +1,29 @@
 browser = chrome
 token   = eyJhbGciOiJIUzUxMiJ9.eyJzdWIiOiJhZG1pbiIsImF1dGgiOiJST0xFX0FETUlOLFJPTEVfVVNFUiIsImV4cCI6MTYwNDYxNzgyM30.6pYwvDKGECkHg6dx5Z__mYZofyHGutdyAw42IRaDNwiyxNtFWUmpTimfh1r3DklSrYqkXUaSVikcWH_ooBd5UA
-api_url = https://www.gmibank.com/api/tp-account-registrations
\ No newline at end of file
+api_url = https://www.gmibank.com/api/tp-account-registrations
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+singin_url =  https://gmibank.com/login
+signin_falseUsername = ramazanBayram
+signin_trueUsername = tataramazan
+signin_falsePassword = Ramazan.1453
+signin_truePassword = Gmi.123
+
+
Index: src/test/java/pages/SigninPage.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package pages;\n\npublic class SigninPage {\n}\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/test/java/pages/SigninPage.java	(revision 8177804d26826c245f3356780f45575d7ecba8b4)
+++ src/test/java/pages/SigninPage.java	(date 1605080574146)
@@ -1,4 +1,25 @@
 package pages;
 
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import utilities.ConfigurationReader;
+import utilities.Driver;
+
 public class SigninPage {
+
+    public SigninPage(){
+        PageFactory.initElements (Driver.getDriver (),this);
+    }
+
+    @FindBy(id="username")
+    public WebElement username;
+    @FindBy(id="password")
+    public WebElement password;
+    @FindBy(xpath = "//*[@id=\"login-page\"]/div/form/div[3]/button[2]/span")
+    public WebElement signin;
+    @FindBy(xpath = "//*[@id=\"login-page\"]/div/form/div[2]/div[1]/div[1]/div/span/text()")
+    public WebElement errorMessaj;
+
+    //
 }
Index: src/test/java/stepdefinitions/SignInNegativeStepDef.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/test/java/stepdefinitions/SignInNegativeStepDef.java	(date 1605080800695)
+++ src/test/java/stepdefinitions/SignInNegativeStepDef.java	(date 1605080800695)
@@ -0,0 +1,63 @@
+package stepdefinitions;
+
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+
+
+import org.junit.Assert;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebElement;
+import pages.SigninPage;
+import utilities.ConfigurationReader;
+import utilities.Driver;
+
+import java.security.Key;
+
+public class SignInNegativeStepDef {
+    SigninPage signinPage = new SigninPage ();
+
+
+
+    @Given("user is on “https:\\/\\/gmibank.com\\/login” page.")
+    public void user_is_on_https_gmibank_com_login_page() {
+    Driver.getDriver ().get (ConfigurationReader.getProperty ("singin_url"));
+    }
+
+    @Given("the user clicks on Username.")
+    public void the_user_clicks_on_Username() {
+        signinPage.username.sendKeys (ConfigurationReader.getProperty ("signin_falseUsername")+ Keys.ENTER);
+        signinPage.password.sendKeys (ConfigurationReader.getProperty ("signin_truePassword")+ Keys.ENTER);
+        signinPage.signin.click ();
+        Assert.assertTrue(signinPage.errorMessaj.getText().contains ("Please check your credentials and try again."));
+
+    }
+    @Given("the user enters the wrong Username")
+    public void the_user_enters_the_wrong_Username() {
+       // signinPage.password.sendKeys (ConfigurationReader.getProperty ("signin_pozitifPassword")+ Keys.ENTER);
+
+    }
+
+    @Given("the user clicks on Password..")
+    public void the_user_clicks_on_Password() {
+        // Write code here that turns the phrase above into concrete actions
+        throw new io.cucumber.java.PendingException();
+    }
+
+    @Given("the user enters the correct Password.")
+    public void the_user_enters_the_correct_Password() {
+        // Write code here that turns the phrase above into concrete actions
+        throw new io.cucumber.java.PendingException();
+    }
+
+    @Given("the user clicks on Sign in .")
+    public void the_user_clicks_on_Sign_in() {
+        // Write code here that turns the phrase above into concrete actions
+        throw new io.cucumber.java.PendingException();
+    }
+
+    @Then("the error message on the screen is confirmed.")
+    public void the_error_message_on_the_screen_is_confirmed() {
+        // Write code here that turns the phrase above into concrete actions
+        throw new io.cucumber.java.PendingException();
+    }
+}
Index: src/test/resources/features/signInNegative.feature
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/test/resources/features/signInNegative.feature	(date 1604932668705)
+++ src/test/resources/features/signInNegative.feature	(date 1604932668705)
@@ -0,0 +1,15 @@
+
+@US005TC01
+  Feature: SignIn NegativeTest
+
+
+
+
+Scenario: US005TC01 User wants to login with invalid username on Sign In.
+  Given user is on “https://gmibank.com/login” page.
+  And the user clicks on Username.
+  And the user enters the wrong Username
+  And the user clicks on Password..
+  And the user enters the correct Password.
+  And the user clicks on Sign in .
+  Then the error message on the screen is confirmed.
\ No newline at end of file
